RecoPixelVertexing/PixelTriplets/test/pixHits.ipynb

```diff
@@ -0,0 +1,2643 @@
+{
```

Comment:
@VinInn this file is almost 1 MB.
I guess it can be useful, but do we normally keep these in the release ?

Reply:
An easy solution to save space is to keep just the source, w/o the histos and results.
The advantage of the full notebook is that you see also the results.
(I admit I shall add a bit nof textual doc)
It is common to find notebook in git.
I already committed one for the Cluster shape analysis. It was also used By D.L. as an example of use of notebook to document analysis in releases at CHEP.

I personally try to encourage to commit for instance the Keras Notebook used to train NN.
Again, they can be just code, or include results (plots) as well. (useful I think if training took weeks)

Reply:
Sounds good to me - I guess somebody else can review this when we try to merge upstream.